cluster/manager/manager.go

```diff
@@ -1223,7 +1223,7 @@ func (c *ClusterManager) StartWithConfiguration(
 		return err
 	}
 
-	c.startClusterDBWatch(lastIndex, kv)
+	_ = c.startClusterDBWatch(lastIndex, kv)
```

Comment:
this change is not needed right?

Reply:
correct, my editor was complaining on unhandled error :)